src/components/rectArticleSummary/Index.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import {
+  Card,
+  Row,
+  Col
+} from 'react-bootstrap';
+import PropTypes from 'prop-types';
+
+const RectArticleSummary = (props) => {
+  const { header, name, time } = props;
+  const date = new Date(time).getTime();
+  const now = new Date().getTime();
+  const hrs = Math.floor((now - date) / 1000 / 60 / 60);
```

Comment:
@olajide1234 we should definitely extract this and use it to replace everywhere it's being used in the app